docker/pytorch/build_matrix.sh

```diff
@@ -15,3 +15,4 @@ echo "TAG='mosaicml/pytorch:1.11.0_cu113-python3.9-ubuntu20.04' BASE_IMAGE='nvid
 echo "TAG='mosaicml/pytorch:1.11.0_cpu-python3.9-ubuntu20.04' BASE_IMAGE='ubuntu:20.04' PYTHON_VERSION='3.9' CUDA_VERSION_TAG='cpu' CUDA_VERSION='cpu' LINUX_DISTRO='ubuntu:20.04' PYTORCH_VERSION='1.11.0' TORCHVISION_VERSION='0.12.0' TARGET='pytorch_stage'"
 # MMCV_TORCH_VERSION for the link https://download.openmmlab.com/mmcv/dist/${CUDA_VERSION_TAG}/${MMCV_TORCH_VERSION}/index.html
 echo "TAG='mosaicml/pytorch_vision:1.10.0_cu113-python3.9-ubuntu20.04' TAG='mosaicml/pytorch_vision:latest' BASE_IMAGE='nvidia/cuda:11.3.1-cudnn8-devel-ubuntu20.04' PYTHON_VERSION='3.9' CUDA_VERSION_TAG='cu113' CUDA_VERSION='11.3.1' LINUX_DISTRO='ubuntu:20.04' PYTORCH_VERSION='1.10.0' TORCHVISION_VERSION='0.11.1' TARGET='vision_stage' MMCV_TORCH_VERSION=torch1.10.0"
+echo "TAG='mosaicml/pytorch_vision:1.11.0_cu113-python3.9-ubuntu20.04' BASE_IMAGE='nvidia/cuda:11.3.1-cudnn8-devel-ubuntu20.04' PYTHON_VERSION='3.9' CUDA_VERSION_TAG='cu113' CUDA_VERSION='11.3.1' LINUX_DISTRO='ubuntu:20.04' PYTORCH_VERSION='1.11.0' TORCHVISION_VERSION='0.12.0' TARGET='vision_stage' MMCV_TORCH_VERSION=torch1.11.0"
```

Comment:
Can we also add a CPU-based and GPU-based non-vision pytorch 1.11 image? Would like to eventually run Jenkins tests on 1.11 (and remove the 1.9 tests).

Reply:
Already done as part of #868

Reply:
Wow, I'm blind. Sorry!